test/test_nn.py

```diff
@@ -4653,6 +4653,19 @@ def multilabelmarginloss_no_reduce_test():
         pickle=False)
 
 
+def softmarginloss_no_reduce_test():
+    t = Variable(torch.randn(10))
```

Comment:
You should probably test something that has more dimensions, esp because the docs advertise this loss as being able to take an input of arbitrary shape.